src/main/java/seedu/address/logic/commands/DeletePersonCommand.java

```diff
@@ -13,23 +13,26 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its displayed index from the contact list of the application.
  */
 public class DeletePersonCommand extends Command {
 
     public static final String COMMAND_WORD = "deletePerson";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the person based on the index number in the contact list displayed.\n"
```

Comment:
```suggestion
            + ": Deletes the person based on the index number in the person list displayed.\n"
```
I think we should use the word 'persons' instead of contact because our application no longer holds just person contacts but events as well.